python/bindings/include/openravepy/openravepy_trajectorybase.h

```diff
@@ -56,6 +56,10 @@ class OPENRAVEPY_API PyTrajectoryBase : public PyInterfaceBase
 
     object SamplePointsSameDeltaTime2D(dReal deltatime, bool ensureLastPoint, PyConfigurationSpecificationPtr pyspec) const;
 
+    object SampleRangeSameDeltaTime2D(dReal deltatime, dReal startTime, dReal stopTime, bool ensureLastPoint) const;
+
+    object SampleRangeSameDeltaTime2D(dReal deltatime, dReal startTime, dReal stopTime, bool ensureLastPoint, PyConfigurationSpecificationPtr pyspec) const;
```

Comment:
This is mostly because all the other methods such as `Insert` and `Sample` are using `PyConfigurationSpecificationPtr` as well, and I followed the same pattern. If we were to change this new api to use non-pointer (`PyConfigurationSpecification`), then all others should change at the same time for consistency?